changelog/unreleased/change-remove-ocs-user

```diff
@@ -0,0 +1,8 @@
+Change: Remove ocs user
+
+BREAKING CHANGE for developers: The user from the ocs api has been removed in favor of the graph user.
```

Comment:
Please add a hint to the breaking change notice, that the underlying data structure of the `user` has changed.